src/MyElefant.php

```diff
@@ -29,13 +29,31 @@ class MyElefant
      */
     private $info;
 
-
     /**
      * Token
      * @var string
      */
     private $token;
 
+    /**
+     * SecretKey
+     * @var string
+     */
+    private $secretKey;
+
+    /**
+     * Flag to prevent permanent looping on token regeneration
+     * @var integer
+     */
+    private $tokenFlag = 0;
+
+    /**
+     * guzzleCatcher Option to Enable/Disable throwing exceptions
+     * on an HTTP protocol errors (i.e., 4xx and 5xx responses).
+     * @var boolean
+     */
+    private $guzzleCatcher = false;
```

Comment:
Ideally, boolean variable names should answer YES/NO questions, ex: isGuzzleEnabled, areErrorsDisplayed, hasNotified, etc